server/app/filters/DisableCachingFilter.java

```diff
@@ -11,8 +12,14 @@
 public class DisableCachingFilter extends EssentialFilter {
   private final Executor exec;
 
+  private final ImmutableList<String> staticAssetPaths =
```

Comment:
suggestion (blocking): make it a static constant
```suggestion
  private static final ImmutableList<String> STATIC_ASSET_PATHS =
```